README.md

```diff
@@ -1 +1,148 @@
-# javascript-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Todo List Project](#-Javascript-project)
+  - [🛠 Built With ](#-built-with-)
+  - [🚀 Live Demo video ](https://youtu.be/sgqM34DgA4Q)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Javascript Capstone<a name="about-project"></a>
+
+**javascript Capstone** 
+The JavaScript capstone project  is about building your own web application based on an external API. We select an API (The pokemon api) that provides data about our selected api and we build  the webapp around it. The webapp have 3 user interfaces.
+
+
+## 🛠 Built With <a name="built-with"></a>
+    HTML
+    CSS
+    Javascript(ES6)
+    Webpack
+    AAA patern
+    jest Tests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ - [Live Demo video](https://youtu.be/sgqM34DgA4Q)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- Create a local directory that you want to clone the repository.
+
```

Comment:
- [ ] Please make sure that the reservation menu is responsive look how  does it look like when the user uses a mobile it doesn't fill the whole screen 
![image](https://user-images.githubusercontent.com/74744787/211160115-25f93a75-e0dc-4cbf-80e9-095aa10bcf16.png)

- [Optional ] make sure to create a validation for the reservation and look at how the user can create two dates with the second one before the first one. 
![image](https://user-images.githubusercontent.com/74744787/211160234-5dad7717-ed6a-4e8b-b771-65905899d696.png)

- [ ] Good Job by creating the reservation data inside a `ul` tag but I think it would be better to use a `ul` tag for the anime data too.
![image](https://user-images.githubusercontent.com/74744787/211160314-476871dd-6030-4dd7-8825-499b06d163ff.png)
-  [ Optional] Please create a validation for the name length, so the user can't put a very long name.
![image](https://user-images.githubusercontent.com/74744787/211160380-f3df021b-92dd-488c-8a26-4b22bcae2b8a.png)

Reply:
@Hassaanjbaig-code I marked all comments about responsiveness as optional, but I really suggest you make it responsive.

Reply:
@MeqdamAlqudah , thank you, we will surely implement responsiveness but for now, we are running out of time and need to submit. Thank you for your suggestion